ext/RMagick/rmpixel.c

```diff
@@ -602,7 +639,11 @@ Pixel_from_hsla(int argc, VALUE *argv, VALUE class ATTRIBUTE_UNUSED)
 
     exception = AcquireExceptionInfo();
 
+#if defined(IMAGEMAGICK_7)
+    (void) QueryColorCompliance(name, AllCompliance, &pp, exception);
```

Comment:
Could we could alias `QueryColorCompliance` to `QueryMagickColor` for IM6?

Reply:
An alias would no work because IM7 has an extra argument.

Reply:
A sorry, I missed the extra argument.